payments-core/src/main/java/com/stripe/android/googlepay/GooglePayConfig.kt

```diff
@@ -0,0 +1,44 @@
+package com.stripe.android.googlepay
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+internal data class GooglePayConfig(
```

Comment:
This class name collides with `com.stripe.android.GooglePayConfig`, now that we're on it, should we rename it?

Reply:
Renamed this class to `GooglePaySheetConfig`